package.json

```diff
@@ -146,6 +146,7 @@
     "got": "9.6.0",
     "gscan": "4.31.2",
     "html-to-text": "8.2.0",
+    "human-number": "^2.0.0",
```

Comment:
Would you be able to pin the dependency to `2.0.0`?

Reply:
Done!